Fix #123 - Update to Placeholder Images topic (#130)
magento/merchdocs: src/catalog/product-image-placeholders.md

Magento uses temporary images as placeholders until the permanent product images become available. A different placeholder can be uploaded for each role. The initial placeholder image is the Magento logo, which you can replace with an image of your choice.

![]({% link images/images/storefront-image-placeholder.png %}){: .zoom}
_Image Placeholder_

> ## Upload Placeholder Images:

Reviewer: remove the colon (:) - linting rule will flag that

1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.

> 1. In the panel on the left under **Catalog**, choose **Catalog**.

Reviewer: ```suggestion
1. In the left panel, expand **Catalog** and choose **Catalog** underneath.
```

1. Expand ![Expansion icon]({% link images/images/btn-expand.png %}){: .Inline} the **Product Image Placeholders** section.

> 1. For each image role, click <span class="btn"> Choose File </span>. Find the image on your computer and upload the file. You can use the same image for all three roles, or you can upload a different placeholder image for each role.

Reviewer: remove spaces in button span tag

Author: done

    ![]({% link images/images/config-catalog-catalog-product-image-placeholders.png %}){: .zoom}  
    _[Product Image Placeholders]({% link configuration/catalog/catalog.md %})_

> 1. When complete, click <span class="btn"> Save </span>.

Reviewer: remove spaces in button span tag

Author: removed in 2 places
